stylus/include/AboutUs.styl

```diff
@@ -0,0 +1,52 @@
+.AboutUs
+	>div:nth-child(1)
+		display: flex
+		padding-bottom: 6rem
+		padding-top: 4rem
+		>div
+			width: 50%
```

Comment:
Children of flex should use

```flex:${numberValue}```

not hardcoded widths

Reply:
Make sure to change this to `flex: 1`. As Chris said, the width is hardcoded, so if you ever added another element, 50% would not be appropriate. Flexbox keeps things dynamic based on ratios (in this case, 1:1).

Reply:
Make sure you don't forget this! `width: 50%` should be `flex: 1`.